### ISSUE TITLE
New work record for

### ISSUE THREAD

**Issue by Ralph-Lee-UK** [BetaMasaheft/Documentation#1694]
EMIP 2338 is a treatise in Amharic on the Passion of the Cross. The work is quite extensive and it seems to me that it would deserve a work file? Any thoughts?
![EMIP02339__Weiner307_004](https://user-images.githubusercontent.com/39619238/107361024-a17c0980-6ace-11eb-856e-cf82b3be7338.jpg)

**Comment by DenisNosnitsin1970** [comment]
It looks like it deserves an ID, sure.

**Comment by DenisNosnitsin1970** [comment]
The topic ሕማማተ፡ መስቀል፡ is quite extensive.

**Comment by Ralph-Lee-UK** [comment]
@DenisNosnitsin1970  yes indeed - i did wonder whether this is related to Ge'ez works with the same or similar title.

**Comment by DenisNosnitsin1970** [comment]
The same work appears in MS ESmr041, fols. 117ra-161vb. It is in BM, but the description is not adjusted yet and the images are not made accessible. Can you open them, Pietro?

**Comment by DenisNosnitsin1970** [comment]
Ralph, the Ms ist ESmr05, fols. 117ra-161vb (not ESmr041). I apologize.

**Comment by eu-genia** [comment]
the images for MR-005 are now accessible

**Comment by Ralph-Lee-UK** [comment]
@DenisNosnitsin1970 - just checking ESmr05, fols. 117ra-161vb is a Gǝʿǝz text and 117ra seems to be in the middle of a text - can you confirm which one of these I should be looking at? Apologies!

**Comment by DenisNosnitsin1970** [comment]
My fault, there was a confusion. The ms is ESmr45, also available in BM, the folia numbers are correct.
